bake-kdf/README.md

```diff
@@ -35,6 +35,6 @@ assert_eq!(key, hex!("bbd7ece0080bee33c776a140f8d807a113a119a4e4d4270f9f2018fbd5
 [build-image]: https://github.com/RustCrypto/KDFs/actions/workflows/bake-kdf.yml/badge.svg
 [build-link]: https://github.com/RustCrypto/KDFs/actions/workflows/bake-kdf.yml
 [license-image]: https://img.shields.io/badge/license-Apache2.0/MIT-blue.svg
-[rustc-image]: https://img.shields.io/badge/rustc-1.41+-blue.svg
```

Comment:
Add licensing info similarly to the other crates.

Reply:
You haven't resolved it, no? I could do it in a separate PR, but then at least write that you disagree with this comment.

Reply:
Yeah, PEBKAC